### ISSUE TITLE
Multiple-line equations not displayed properly

### ISSUE THREAD

**Issue by heytitle** [glm-tools/pyglmnet#370]
Hi,

There are several multiple-line equations, and some of them aren't written with LaTex's `aligned` environment.

The figure below is [such an instance](http://glm-tools.github.io/pyglmnet/tutorial.html#cyclical-coordinate-descent). 
<img width="595" alt="Screen Shot 2020-02-28 at 9 44 45 AM" src="https://user-images.githubusercontent.com/1214890/75524937-2f222e80-5a0f-11ea-9cf9-39fd08a77493.png">
Looking at the source, the problem seems to be caused by an additional space between `&` and ` `.

```
 \begin{eqnarray}
    \beta_k^{t} & = \beta_k^{t-1} - (h_k^{t-1})^{-1} \tilde{g}_k^{t-1} \\
    \beta_j^{t} & = \beta_j^{t-1}, \forall j \neq k
\end{eqnarray}
```

I guess removing these spaces would solve the problem.

Pages that I find this issue:
- http://glm-tools.github.io/pyglmnet/tutorial.html
- http://glm-tools.github.io/pyglmnet/cheatsheet.html

**Comment by pavanramkumar** [comment]
thanks for reporting! we accept pull requests :)

**Comment by jasmainak** [comment]
closed by #371
